src/components/study-pane.js

```diff
@@ -539,13 +576,40 @@ const StudyPane = (props) => {
             }
         }
 
-        const sldTitle =
-            useName && displayedVoltageLevel
+        let sldTitle;
+        let svgUrl;
+        if (displayedVoltageLevel) {
+            sldTitle = useName
                 ? displayedVoltageLevel.name +
                   ' \u002D ' +
                   network.getSubstation(displayedVoltageLevel.substationId)
                       .countryName
                 : displayedVoltageLevelId;
```

Comment:
If the id is not present the tittle is 'undefined'

Reply:
Corrected